core/presentation/designsystem/src/main/java/com/nolawiworkineh/designsystem/components/ProfileInitialButton.kt

```diff
@@ -0,0 +1,67 @@
+package com.nolawiworkineh.designsystem.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.nolawiworkineh.designsystem.Theme.TaskyGray
+import com.nolawiworkineh.designsystem.Theme.TaskyLightGray
+import com.nolawiworkineh.designsystem.Theme.TaskyTheme
+
+@Composable
+fun ProfileIcon(
+    modifier: Modifier = Modifier,
+    initials: String,
+    onProfileClick: () -> Unit,
+    ) {
```

Comment:
When creating composables, keep the following in mind:

1. `Modifier` should be the first _optional_ parameter in your parameter list. 
2. The last parameter should not be a non-Compose lambda.
3. If a Composable is only used in this file, mark it as `private`.

Not following **no 2**, you could write the following, which would be misleading as someone would expect this lambda block to be for more Compose code:

```
@Composable
fun MyCustomComposable(
  modifier: Modifier = Modifier,
  onClick: () -> Unit
)

// Usage
MyCustomComposable(
  modifier = Modifier.size(12.dp),
) {
  // WRONG - handle onClick here
}
```